src/index.css

```diff
@@ -33,6 +35,44 @@ body {
   }
 }
 
+::-webkit-scrollbar {
+  width: 25px;
+}
+::-webkit-scrollbar-track {
+  background: #00000000;
+  background: linear-gradient(
+    90deg,
+    rgba(0, 0, 0, 0) 47%,
+    #fed823 47%,
+    #fed823 51%,
+    rgba(0, 0, 0, 0) 52%
+  );
+}
+::-webkit-scrollbar-thumb {
+  background: #fed823;
+  border-radius: 20px;
+  background-clip: content-box;
+  border: 8px solid transparent;
+  box-shadow: inset #fed823;
+}
+::-webkit-scrollbar-thumb:hover {
+  background: #fed950;
+  border-radius: 20px;
+  background-clip: content-box;
+  border: 7.5px solid transparent;
```

Comment:
I think this looks fine

https://user-images.githubusercontent.com/49658988/176898450-69e6c1b6-e04f-4c15-8752-dbbc3af43043.mp4

e